config.json

```diff
@@ -1,6 +1,7 @@
 {
   "language": "Julia",
   "active": false,
+  "blurb": "Julia is an open-source high-level, high-performance dynamic programming language designed for technical and scientific computing while also being effective for general-purpose tasks. It is convenient to use for daily work but also runs fast enough to be deployed for high-performance applications.",
```

Comment:
It's a little bit on the long side for the blurb—we're aiming at about 30-ish words (this is around 40).

How about moving the high-performance + general purpose out of the first sentence, and focusing the second sentence on those two things?

> Julia is an open-source high-level, dynamic programming language whose sweet spot is technical and scientific computing. It is convenient for day-to-day work and packs a high-performance punch.